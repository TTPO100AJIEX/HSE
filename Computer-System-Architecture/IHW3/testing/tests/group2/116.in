-0.4777634660052579