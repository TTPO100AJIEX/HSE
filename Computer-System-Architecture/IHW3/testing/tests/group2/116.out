-0.4981070560016018